src/processing/dom/index.js

```diff
@@ -216,6 +216,11 @@ export default class DomProcessor {
         this.adapter.setAttr(el, 'autocomplete', 'off');
     }
 
+    _processContentSecurityPolicy (el) {
+        if (this.adapter.getAttr(el, 'http-equiv').toLowerCase() === 'content-security-policy')
+            this.adapter.removeElement(el);
```

Comment:
Maybe we should remove meta attributes instead, so it will be just skiped by the browser?